src/nvim/ui.coffee

```diff
@@ -2,6 +2,7 @@
 # handle redraw events from neovim
 # Copyright (c) 2015 Lu Wang <coolwanglu@gmail.com>
 
+punycode = require 'punycode'
```

Comment:
I wasn't sure if this was necessary since `punycode` is included in `node.js` (and thus atom-shell) by default. I can add it though